x/compute/module_test.go

```diff
@@ -189,11 +187,11 @@ func TestHandleInstantiate(t *testing.T) {
 
 	assertContractList(t, q, data.ctx, 1, []string{contractAddr.String()})
 	assertContractInfo(t, q, data.ctx, contractAddr, 1, creator)
-	assertContractState(t, q, data.ctx, contractAddr, state{
-		Verifier:    []byte(fred),
-		Beneficiary: []byte(bob),
-		Funder:      []byte(creator),
-	})
+	// assertContractState(t, q, data.ctx, contractAddr, state{
+	// 	Verifier:    []byte(fred),
+	// 	Beneficiary: []byte(bob),
+	// 	Funder:      []byte(creator),
+	// })
```

Comment:
why did you comment stuff out in this file?

Reply:
It's a remnant of #387...

Reply:
Oh I see, very well :+1: